R/download.R

```diff
@@ -4,6 +4,75 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
+#' Download Bugzilla issues using the Bugzilla REST API
+#'
+#' Downloads bugzilla issues and returns a list of bugzilla bug ids for use with
+#'  \code{\link{download_bugzilla_comments_from_rest_api}}
+#'
+#' @param bugzilla_site URL to specific bugzilla site
+#' @param start_timestamp when to start bug retrieval (ex. 2023-01-01T00:14:57Z)
+#' @param save_folder_path the full *folder* path where the bugzilla issues will be stored
+#' @seealso \code{\link{download_bugzilla_comments_from_rest_api}}
+#' @export
+download_bugzilla_issues_from_rest_api <- function(bugzilla_site, start_timestamp, save_folder_path){
+  # Format link to retrieve data using Bugzilla REST API
+  bugzilla_site <- paste(bugzilla_site, "/rest", sep="")
+
+  # Make sure folder path is correctly formatted
+  if (stringr::str_sub(save_folder_path,-1) != "/"){
```

Comment:
Kaiaulu rely on the stringr library. Could you replace this by stringi library? There should be a function equivalent to this one.